src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
-// possible values for the `key` property on error messages we get from the server
-const ACCOUNT_FLAGGED_FOR_FRAUD_KEY = 'cnp.payment.flashes.on.record.message';
-const GENERIC_ERROR_KEY = 'cnp.payment.generic.error.message';
-const INVALID_ROUTING_NUMBER_KEY =
-  'payment.accountRoutingNumber.invalidCheckSum';
-const PAYMENT_RESTRICTIONS_PRESENT_KEY =
-  'payment.restriction.indicators.present';
-const ROUTING_NUMBER_FLAGGED_FOR_FRAUD_KEY =
-  'cnp.payment.routing.number.fraud.message';
+import {
+  hasFlaggedForFraudError,
```

Comment:
These functions used to live directly in this file but have been moved to the `util/index.js` so that the new `createEventDataObjectWithErrors` helper can use the functions as well.